code/modules/power/lighting.dm

```diff
@@ -722,16 +793,17 @@
 	if(istype(I, /obj/item/reagent_containers/syringe))
 		var/obj/item/reagent_containers/syringe/S = I
 
-		to_chat(user, "You inject the solution into the [src].")
+		if(!length(S.reagents.reagent_list))
+			return
 
+		to_chat(user, "<span class='notice'>You inject the solution into [src].</span>")
```

Comment:
This one combined with 806 is still a bit odd. Here it says you did it. Then if you don't have plasma in the syringe it won't clear it.
I'd say put this one inside the if scope and maybe give a warning message saying that you can't rig it or such if you don't have plasma in the syringe

Reply:
Damn, could've sworn I fixed that.